### PR TITLE
fix: Links to localhost addresses

### PR DESCRIPTION
**Pull request by nschonni**
### Why:

Rather than ignoring localhost links in the linter, it's better to just not directly link to them

### What's being changed:

Change localhost links to code blocks

### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/developers/apps/authorizing-oauth-apps.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ The optional `redirect_uri` parameter can also be used for localhost URLs. If th
 
 For the `http://localhost/path` callback URL, you can use this `redirect_uri`:
 
-   http://localhost:1234/path
+    http://localhost:1234/path
```
Comment (nschonni) [inline content/developers/apps/authorizing-oauth-apps.md:275]: I'm not sure if the code block is the best formatting given the previous line, but it seems like that was the intention here but was missing a space

**Comment by janiceilene** [comment]
@nschonni Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by heiskr — APPROVED** [review]
Makes sense to me

File: content/developers/apps/authorizing-oauth-apps.md (hunk 1 above)
Comment (lucascosti) [reply content/developers/apps/authorizing-oauth-apps.md:275]: You're probably right here, but while where at it, let's change this to be our usual code block style:

````suggestion
```
http://localhost:1234/path
```
````

**Review by lucascosti — APPROVED** [review]
Looks great, @nschonni, thanks for this change! I'll just commit the one suggestion and then merge this in 🚀

**Pull request merged**